x-pack/plugins/infra/public/translations.ts

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
I created this file as a starting point to put our translations here. I will create an another ticket to move all translations here and use accordingly